packages/synapse-react-client/src/lib/containers/personal_access_token/CreateAccessTokenModal.tsx

```diff
@@ -85,118 +77,82 @@ export const CreateAccessTokenModal: React.FunctionComponent<
     }
   }
 
+  const dialogContent = showCreatedToken ? (
+    <>
+      <span className="SRC-boldText">
+        This token will not be able to be retrieved again.
+      </span>{' '}
+      <span>
+        If needed, generate a new personal access token, and delete this one.
+      </span>
+      <div className="AccessTokenModal__CopyToClipboardContainer">
+        <CopyToClipboardInput value={createdToken} inputWidth={'350px'} />
+      </div>
+      <p>
+        This token grants access to your account functions and should be treated
+        like a password.
+      </p>
+    </>
+  ) : (
+    <div className="SRC-marginFive">
+      <TextField
+        autoFocus
+        label="Token Name"
+        value={tokenName}
+        onChange={handleTokenNameChange}
+        placeholder="e.g. Synapse command line access on my laptop"
+      />
+      <div className="AccessTokenModal SRC-marginBottomTop">
+        <FormLabel className="SRC-boldText">Token Permissions</FormLabel>
+        <Checkbox
+          label={scopeDescriptions.view.displayName}
+          checked={viewAccess}
+          onChange={() => setViewAccess(!viewAccess)}
+        >
+          <div className="AccessTokenModal__ScopeDescription">
+            {scopeDescriptions.view.description}. Required to use Synapse
+            programmatic clients.
+          </div>
+        </Checkbox>
+        <Checkbox
+          label={scopeDescriptions.download.displayName}
+          checked={downloadAccess}
+          onChange={() => setDownloadAccess(!downloadAccess)}
+        >
+          <div className="AccessTokenModal__ScopeDescription">
+            {scopeDescriptions.download.description}
+          </div>
+        </Checkbox>
+        <Checkbox
+          label={scopeDescriptions.modify.displayName}
+          checked={modifyAccess}
+          onChange={() => setModifyAccess(!modifyAccess)}
+        >
+          <div className="AccessTokenModal__ScopeDescription">
+            {scopeDescriptions.modify.description}
+          </div>
+        </Checkbox>
+      </div>
+      {showErrorMessage && <ErrorBanner error={errorMessage}></ErrorBanner>}
```

Comment:
Remove surrounding div (`<div className="SRC-center-text">`) - text is already centered, so not needed